src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,14 @@
 package mate.academy.service;
 
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+
     public boolean login(String email, String password) {
-        return false;
+        if (UserService.findByEmail(email) == null) {
```

Comment:
let's create a UserService object and call method on this object

and please use local variable for user returned by findByEmail() method